lib/api/controllers/securityController.js

```diff
@@ -229,8 +252,8 @@ class SecurityController {
 
     return this.kuzzle.repositories.profile.loadMultiFromDatabase(request.input.body.ids)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 89 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)